### ISSUE TITLE
Mod Installing Window Freezes

### ISSUE THREAD

**Issue by superloach** [ATLauncher/ATLauncher#596]
**Describe the bug**
With certain Modrinth mods, the "Installing" window freezes up.

**To Reproduce**
On a 1.18.2 instance, try installing the "AutoSwitch" mod from Modrinth. The "Installing" window freezes up, and the only available button is "Cancel".

**Expected behavior**
The mod info properly loads, and I am able to click "Add".

**Logs**
```
[04/06/2022 21:37:18 pm] java.lang.NullPointerException: Cannot invoke "String.replace(java.lang.CharSequence, java.lang.CharSequence)" because "projectId" is null
	at com.atlauncher.utils.ModrinthApi.getProject(ModrinthApi.java:120)
	at com.atlauncher.gui.card.ModrinthProjectDependencyCard.setupComponents(ModrinthProjectDependencyCard.java:64)
	at com.atlauncher.gui.card.ModrinthProjectDependencyCard.<init>(ModrinthProjectDependencyCard.java:60)
	at com.atlauncher.gui.dialogs.ModrinthVersionSelectorDialog.lambda$reloadDependenciesPanel$2(ModrinthVersionSelectorDialog.java:129)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at com.atlauncher.gui.dialogs.ModrinthVersionSelectorDialog.reloadDependenciesPanel(ModrinthVersionSelectorDialog.java:129)
	at com.atlauncher.gui.dialogs.ModrinthVersionSelectorDialog.lambda$setupComponents$7(ModrinthVersionSelectorDialog.java:237)
	at java.desktop/javax.swing.JComboBox.fireActionEvent(JComboBox.java:1294)
	at java.desktop/javax.swing.JComboBox.contentsChanged(JComboBox.java:1367)
	at java.desktop/javax.swing.AbstractListModel.fireContentsChanged(AbstractListModel.java:127)
	at java.desktop/javax.swing.DefaultComboBoxModel.setSelectedItem(DefaultComboBoxModel.java:94)
	at java.desktop/javax.swing.DefaultComboBoxModel.addElement(DefaultComboBoxModel.java:132)
	at java.desktop/javax.swing.JComboBox.addItem(JComboBox.java:746)
	at com.atlauncher.gui.dialogs.ModrinthVersionSelectorDialog.lambda$null$15(ModrinthVersionSelectorDialog.java:297)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at com.atlauncher.gui.dialogs.ModrinthVersionSelectorDialog.lambda$getFiles$17(ModrinthVersionSelectorDialog.java:297)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

**Screenshots**
![image](https://user-images.githubusercontent.com/30514424/172031466-7ec2a65c-27b4-4405-b6eb-31dbe6ce56a2.png)

**System information (please complete the following information, or remove if not applicable):**

- OS (including distro/version): Arch Linux
- Java Version: `jdk-temurin 18.0.1.u10-1`

**Additional context**
Seems that at some point the Modrinth project ID is null - *not sure if it should be* - but it stops the UI rendering.

**Comment by RyanTheAllmighty** [comment]
Yup this was fixed by version [3.4.16.1](https://github.com/ATLauncher/ATLauncher/releases/tag/v3.4.16.1), make sure you're up to date.

If you are up to date, then please provide logs using the Upload Logs button on the launcher after the issue occurs to generate a sharable link to your logs

**Comment by superloach** [comment]
Oh, my bad. Closing this now :)
